pkg/issuer/venafi/client/venaficlient_test.go

```diff
@@ -35,6 +35,7 @@ const (
 	zone                = "test-zone"
 	username            = "test-username"
 	password            = "test-password"
+	clientId            = "cert-manager"
```

Comment:
I don't think this ever actually gets used, but I think it should ideally be `cert-manager.io` to match the default.

Reply:
thanks for the review, changed to `defaultClientId`